src/DynamoCore/Linting/Interfaces/IRuleEvaluationResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Linting.Interfaces
+{
+    public interface IRuleEvaluationResult
```

Comment:
since we made the subclasses internal let's have this one as internal as well

Reply:
if i make the interface internal i get issues with the linter rules `InitializeRule` (https://github.com/DynamoDS/Dynamo/pull/11606/files#diff-07729abac4d567c8b6ac2d6adff4f79e04715870506e8098e5cc1f3ca1afbcf0R45) as this will then throw an error because the return type is less accessible than the method.